docs/modular-resources/examples/tflite-module.md

```diff
@@ -269,7 +269,7 @@ With everything configured and running, you can now run the inference client tha
 
    ```sh { class="command-line" data-prompt="$"}
    cd ~/example_workspace/opt/bin
-   ./example_audio_classification_client --model-label-path ~/example_workspace/yamnet_label_list.txt --robot-host my-robot-main.abcdefg123.viam.cloud --robot-secret abcd1234abcd1234abcd1234abcd1234abcd1234abcd1234
+   ./example_audio_classification_client --model-label-path ~/example_workspace/yamnet_label_list.txt --robot-host my-robot-main.abcdefg123.viam.cloud --robot-api-key abcdef1g23hi45jklm6nopqrstu7vwx8 --robot-api-key-id a1234b5c-678d-9012-3e45-67fabc8d9efa
```

Comment:
The `example_audio_classification_client` binary https://github.com/viamrobotics/viam-cpp-sdk/blob/main/src/viam/examples/mlmodel/example_audio_classification_client.cpp hasn't been updated to support API keys. Would it be possible to keep the `docs/modular-resources/examples/tflite-module.md` using secrets?

cc @benjirewis in case you all would be down to update the example code to use API keys

Reply:
Probably yes! I know we just did something similar for the Python SDK. cc @stuqdog what do you think?

Reply:
I have a PR up for that - please do not duplicate that work https://github.com/viamrobotics/viam-cpp-sdk/pull/168

Reply:
Yeah I think we can do that for sure! @npentrel feel free to add me as a reviewer for your PR if you'd like, or loop us (SDK team) in to help with switching to using API keys.

Reply:
So I've just updated that PR and there is a runtime error -- the rustutils don't seem able to handle the api-key credential type. @benjirewis @stuqdog would either of you be able to help with that? I have also pinged @acmorrow who was reviewing that PR for me

Reply:
So I don't think this is a `rust-utils` issue, `rust-utils` doesn't have an opinion on credential type and just passes along whatever it receives. I suspect the error is coming from the server running outdated code that doesn't know about `api-key`s. When did you create your robot host? how are you running it?

Reply:
I created a new robot in the live version of the app. It is possible I didn't update viam-server though, let me try